llama-index-integrations/embeddings/llama-index-embeddings-voyageai/tests/test_voyageai_utils.py

```diff
@@ -0,0 +1,27 @@
+from pytest import MonkeyPatch
+from typing import Any, Dict
+
+from llama_index.core.embeddings.mock_embed_model import MockEmbedding
+from llama_index.core.embeddings.utils import resolve_embed_model
+from llama_index.embeddings.voyageai import VoyageEmbedding
+
+
+def mock_voyageai_embeddings(self: Any, *args: Any, **kwargs: Dict[str, Any]) -> Any:
+    """Mock OpenAIEmbedding."""
+    super(VoyageEmbedding, self).__init__(voyage_api_key="fake")
+    return
+
+
+def test_resolve_embed_model(monkeypatch: MonkeyPatch) -> None:
```

Comment:
What is this test used for? I see openai's package also has it, but couldn't understand.

Reply:
Hm. It tests the `resolve_embed_model` function, but since the same works (and has been tested in the OpenAI package), we can skip the same. Also, this testing is interesting. 

Yes, i copied the same from the OpenAI package.

I can remove the same, or I leave it - but I don't think it makes any difference if I delete it.

Reply:
We can keep it, but I just didn't get what this `resolve_embed_model` function is used for

Reply:
If i see it right, `resolve_embed_model` creates a `BaseEmbedding` instance using the first parameter - which can be either a string (this case `HuggingFaceEmbedding` will be created) or the instance itself.

I don't think we should test the same, i removed it.